libweasyl/libweasyl/models/content.py

```diff
@@ -189,7 +189,7 @@ def set_tags(self, new_tags, done_by_artist=False):
     def create(cls, owner, title, rating, description, category, subtype, folder, tags, friends_only=False,
                critique_requested=False, submission_data=None, cover_data=None, thumbnail_data=None, embed_link=None,
                submission_size_limit=None):
-        from libweasyl.media import fetch_or_create_media_item, make_cover_media_item
+        from libweasyl.media import MediaItem, make_cover_media_item
```

Comment:
`MediaItem` should be imported from `libweasyl.models.media`, or `libweasyl.media` should list `'MediaItem'` in its `__all__` where `'fetch_or_create_media_item'` was.

Reply:
Cleaned this up, and moved a couple of imports that are buried in the code up to the top of the file